docs/extras/modules/model_io/models/chat/integrations/google_vertex_ai.mdx

```diff
@@ -6,10 +6,17 @@ import CodeBlock from "@theme/CodeBlock";
 
 # ChatGoogleVertexAI
 
-The Vertex AI implementation is meant to be used in Node.js and not
-directly from a browser, since it requires a service account to use.
+LangChain.js supports Google Vertex AI chat models as an integration.
+It supports two different methods of authentication based on whether you're running
+in a Node environment or a web environment.
 
-Before running this code, you should make sure the Vertex AI API is
+## Setup
```

Comment:
This (and probably much of the configuration notes) is going to be repetitive between four (so far) modules. Perhaps it would makes sense to move them all into a common page (perhaps something under Guides -> Deployment?) and either include or reference them all in the module documentation?

Reply:
We can factor auth instructions out into a separate .mdx file and import it to avoid repetition in a future PR